docs/files/add-file.md

```diff
@@ -16,10 +16,9 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications find the application where you want to add a configuration file and click `DETAILS`
-1. On the upper-right corner of the applications details view click `SETTINGS`
-1. Use the drop-down menu to choose `CONFIGURATION FILE`
-1. On the right corner of the list click `ADD CONFIGURATION FILE`
+1. In the list of applications, find the application where you want to add a configuration and click the `NAME` of the application
+1. Click the `CONFIGURATION FILES` tab
+1. On the right corner of the applications tab click `MANAGE`
```

Comment:
In cases like this other articles, as [Add Pipeline](https://www.devopness.com/docs/pipelines/add-pipeline/) [1], use the tab name

```suggestion
1. On the right corner of the configuration files tab click `MANAGE`
```

[1]

https://github.com/devopness/devopness/blob/1226718ee38cf9d5ab3ee64e85d334642468e938/docs/pipelines/add-pipeline.md?plain=1#L21

Reply:
Ok, fixed!